lib/provisioning/Provisioner.js

```diff
@@ -212,12 +212,53 @@ Provisioner.prototype._updateBridgingState = Promise.coroutine(
     }
 );
 
+// Utility function for attempting to send a request to a matrix endpoint
+//  that might be rate-limited.
+//
+//  This function will try [attempts] times to apply function [fn] to [obj] by
+//  calling fn.apply(obj, args), with args being any arguments passed to retry
+//  after [fn]. If an error occurs, the same will be tried again after
+//  [retryDelayMs]. If an error err is thrown by [fn] and err.data.retry_after_ms
```

Comment:
`[ ]` is generally used for arrays so this is a bit confusing. Why not use markdown-style ` ?